src/python/grpcio_tests/tests/unit/_metadata_code_details_test.py

```diff
@@ -67,6 +68,9 @@ def unary_unary(self, request, context):
             self._received_client_metadata = context.invocation_metadata()
             context.send_initial_metadata(_SERVER_INITIAL_METADATA)
             context.set_trailing_metadata(_SERVER_TRAILING_METADATA)
+            if self._call_abort:
+                context.abort(self._code, self._details)
+                return
```

Comment:
Why the return statements here and below?

Reply:
I want to ensure we don't accidentally fall through and set_code and set_details causing the test to unintentionally pass.

Reply:
`raise ValueError('Unreachable statement reached in _Servicer!')`? If we were in the test case instance we'd call `unittest.TestCase.fail`, but we're not, so an exception's just as good.

Reply:
The problem is that raising *any* exception will be caught by server logic and we end up with a similar mode that may make our tests pass even if abort didn't do anything. I don't think there's a perfect way (without keeping a TestCase reference in the servicer) to handle it (even returning None is prone to that behavior, but I prefer doing something vastly different than raising (cause that's what `.abort()` does).

Reply:
Hmmm... okay. Thanks for spelling it out.

Reply:
I have though about it a bit more and perhaps the best thing is to do is `return object()` and completing the RPC **successfully**. That should definitely cause a test failure.